app/src/main/java/com/tranhoabinh/framgia/moviedbkotlin/data/repository/RepositoryImpl.kt

```diff
@@ -25,19 +27,24 @@ class RepositoryImpl constructor(
     }
 
 
-    override fun addFavoriteMovie(movie: Movie?): Long {
-        return roomMovieDao.insertMovie(movie)
+    override fun addFavoriteMovie(movie: Movie?) {
+        doAsync {
+            roomMovieDao.insertMovie(movie)
+        }
```

Comment:
sao a k dùng rx ạ. Dùng rx có một lợi thế so với doAsync là có thể handle lỗi

Reply:
a search thì thấy anko cũng có handle error. a đang gặp bug backstack cần fix trước nên phần handle error a sẽ tìm hiểu và thêm vào sau. https://medium.com/@andrea.bresolin/playing-with-kotlin-in-android-coroutines-and-how-to-get-rid-of-the-callback-hell-a96e817c108b